build/buildx/install.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Install Docker Buildx
+description: How to install Docker Buildx
+keywords: build, buildx, buildkit
+---
+
+## Docker Desktop
+
+Docker Buildx is included in [Docker Desktop](../../desktop/index.md) for
+Windows, macOS, and Linux.
+
+## Linux packages
+
+Docker Linux packages also include Docker Buildx when installed using the
+[DEB or RPM packages](../../engine/install/index.md).
+
+## Manual download
+
+> **Important**
+>
+> This section is for unattended installation of the buildx component. These
+> instructions are mostly suitable for testing purposes. We do not recommend
+> installing buildx using manual download in production environments as they
+> will not be updated automatically with security updates.
+>
+> On Windows, macOS, and Linux workstations we recommend that you install
+> [Docker Desktop](../../desktop/index.md) instead. For Linux servers, we recommend
+> that you follow the [instructions specific for your distribution](#linux-packages).
+{: .important}
+
+You can also download the latest binary from the [releases page on GitHub](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"}.
+
+Rename the relevant binary and copy it to the destination matching your OS:
+
+| OS       | Binary name          | Destination folder                         |
+|----------|----------------------|--------------------------------------------|
+| Linux    | `docker-buildx`      | `$HOME/.docker/cli-plugins`                |
+| macOS    | `docker-buildx`      | `$HOME/.docker/cli-plugins`                |
+| Windows  | `docker-buildx.exe`  | `%USERPROFILE%\.docker\cli-plugins`        |
+
+Or copy it into one of these folders for installing it system-wide.
+
+On Unix environments:
+
+* `/usr/local/lib/docker/cli-plugins` OR `/usr/local/libexec/docker/cli-plugins`
+* `/usr/lib/docker/cli-plugins` OR `/usr/libexec/docker/cli-plugins`
+
+On Windows:
+
+* `C:\ProgramData\Docker\cli-plugins`
+* `C:\Program Files\Docker\cli-plugins`
+
+> **Note**
+>
+> On Unix environments, it may also be necessary to make it executable with `chmod +x`:
+> ```shell
+> $ chmod +x ~/.docker/cli-plugins/docker-buildx
+> ```
+
+## Dockerfile
```

Comment:
Could we have this section above "Manual Packages"? It's such a short section, and I think putting it below the manual section means it's more likely to get missed by users (who might otherwise follow the manual installation guidelines).

Reply:
I had some high-level feedback for this PR, but I was in meetings. So, our build doc set first pages have changes since this. I think it poses a good opportunity to change some things.
So on the:
- order or pages: to keep inline "Install" types of topics follow the overview. So I would also move it up.
- structure of buildx: I think we should work on the content as if we already had the features list (simple and advanced use cases ready and you could link from there to here). So, even if, for now, they are going to be short pages. I think we make what users are trying to accomplish more discoverable by 2) Working with Buildx being only the intro (about config) and high-level options 3) promoting the sections to pages (so they show on the primary menu) 4) Check if the names of these pages are the best matches for what users would be searching. 
This will allow us to 1) have the driver for content to be tasks we want to accomplish with Build/x instead of a descriptive/system pov and 2) have room to expand on each of those use cases. 
So
Buildx
\ overview
\ install
\"config"
\working with builders
\multiplatform images

Reply:
I can delay some work and do these changes instead today. Do you mind if I commit to the branch?

Reply:
Can we see that in a follow-up please? This PR is only about moving buildx and preserve the current state.